clientv3/kv.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clientv3
+
+type KV interface {
+	// PUT puts a key-value pair into etcd.
+	// Note that key,value can be plain bytes array and string is
+	// an immutable representation of that bytes array.
+	// To get a string of bytes, do string([]byte(0x10, 0x20)).
+	Put(key, val string) (PutResponse, error)
```

Comment:
Certainly, just curious since the new client API was explicitly designed for that pattern, but as long as that's supported, i think it's great. You coudl always make `Get(key)` a function that takes a KV like `func Get(kv KV, key)` which then wraps the `kv.Do(OpGet(key))`